presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -122,7 +122,7 @@
     public static final String QUERY_MAX_TOTAL_MEMORY_PER_NODE = "query_max_total_memory_per_node";
     public static final String IGNORE_DOWNSTREAM_PREFERENCES = "ignore_downstream_preferences";
     public static final String ITERATIVE_COLUMN_PRUNING = "iterative_rule_based_column_pruning";
-    public static final String FILTERING_SEMI_JOIN_TO_INNER = "rewrite-filtering-semi-join-to-inner-join";
+    public static final String FILTERING_SEMI_JOIN_TO_INNER = "rewrite_filtering_semi_join_to_inner_join";
```

Comment:
we could have a validation that all standard properties eg match `[a-z][a-z0-9]*(_[a-z0-9]+)*`